plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/config/LibraryFactory.java

```diff
@@ -66,6 +69,11 @@ public Library create(IConfigurationElement configurationElement) throws Library
         if (exportString != null) {
           library.setExport(Boolean.parseBoolean(exportString));
         }
+        String recommendationString = configurationElement.getAttribute(ATTR_RECOMMENDATION);
+        if (recommendationString != null) {
+          library.setRecommendation(LibraryRecommendation.valueOf(recommendationString));
```

Comment:
It seems like you also have to do `recommendationString.toUpperCase()`. Otherwise, `valueOf()` fails in my small test.